eclair-core/src/test/scala/fr/acinq/eclair/payment/PaymentInitiatorSpec.scala

```diff
@@ -104,7 +104,7 @@ class PaymentInitiatorSpec extends TestKitBaseClass with FixtureAnyFunSuiteLike
     payFsm.expectMsgType[SendPaymentConfig]
     val tlvs = payFsm.expectMsgType[PaymentLifecycle.SendPayment].finalPayload.records
     assert(tlvs.get[AmountToForward].get.amount == finalAmount)
-    assert(tlvs.get[OutgoingCltv].get.cltv == req.invoice.minFinalCltvExpiryDelta.toCltvExpiry(nodeParams.currentBlockHeight + 1))
+    assert(tlvs.get[OutgoingCltv].get.cltv > req.invoice.minFinalCltvExpiryDelta.toCltvExpiry(nodeParams.currentBlockHeight + 50))
```

Comment:
Where does the 50 come from? Shouldn't it be 100?
```suggestion
    assert(tlvs.get[OutgoingCltv].get.cltv >= req.invoice.minFinalCltvExpiryDelta.toCltvExpiry(nodeParams.currentBlockHeight + 100))
    assert(tlvs.get[OutgoingCltv].get.cltv < req.invoice.minFinalCltvExpiryDelta.toCltvExpiry(nodeParams.currentBlockHeight + 350))
```

Reply:
Right, I wrote the test first and initially used 50, then changed it to 100 in the implementation but not in the tests...
Do you think we should make that configurable by the node operator?
This way they could disable it if they wish to?

Reply:
Making it configurable is a good idea. If everyone uses a delay of at least 100, getting a cltv of 100 is just like getting a cltv of 0 today, so there needs to be the possibility that at least some people are using no delay.

Reply:
I've reworked this in https://github.com/ACINQ/eclair/pull/2469/commits/068619850075f8b36660290188543e39126492ba to make it configurable, which also lets node operators disable it by providing a "delta" containing a single value.

I've created a new `payment` section in `eclair.conf`, we can later migrate some parameters inside that new section (for example, `max-payment-attempts`).